### PR TITLE
Cm azure install linux tentacle

### PR DESCRIPTION
**Pull request by xtreampb**
### Step template checklist

- [ ] `Id` should be a **GUID** that is not `00000000-0000-0000-0000-000000000000`
  - **NOTE** If you are modifying an existing step template, please make sure that you **do not** modify the `Id` property *(updating the `Id` will break the Library sync functionality in Octopus)*. 
- [ ] `Version` should be incremented, otherwise the integration with Octopus won't update the step template correctly
- [ ] Parameter names should not start with `$`
- [ ] **Step template parameter names (the ones declared in the JSON, not the script body) should be prefixed with a namespace so that they are less likely to clash with other user-defined variables in Octopus** (see [this issue](https://github.com/OctopusDeploy/Issues/issues/2126)). For example, use an abbreviated name of the step template or the category of the step template).
- [ ] `LastModifiedBy` field must be present, and (_optionally_) updated with the correct author

Fixes # . _If there is an open issue that this PR fixes add it here, otherwise just remove this line_

### CODE REVIEW

File: step-templates/azure-linux-install-octopus-tentacle.json [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      "Label": "Environments",
+      "HelpText": "Environments to assign this tentacle installation to.<br />\n*Note: Each environment should be on its own line*",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "MultiLineText"
+      }
+    },
+    {
+      "Id": "15150827-7bd3-4146-a798-66344851f602",
+      "Name": "instrallTentacle.octoServerUrl",
```
Comment (harrisonmeister) [inline step-templates/azure-linux-install-octopus-tentacle.json:108]: Could this typo be corrected: `instrall`? (In both the parameter and script)

File: step-templates/azure-linux-install-octopus-tentacle.json [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+      "DefaultValue": "#{Octopus.Web.BaseUrl}",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "9bcd6bb5-2db3-453f-b4f6-6a78f3c39b59",
+      "Name": "installLinuxTentacle.tentacleType",
+      "Label": "Tentacle Type",
+      "HelpText": "Select between a listing or polling tentacle",
```
Comment (harrisonmeister) [inline step-templates/azure-linux-install-octopus-tentacle.json:120]: Real minor typo: `listing` instead of `listening`

**Review by harrisonmeister — CHANGES_REQUESTED** [review]
Couple of minor typos

File: step-templates/azure-linux-install-octopus-tentacle.json (hunk 2 above)
Comment (xtreampb) [reply step-templates/azure-linux-install-octopus-tentacle.json]: addressed in changeset f685690

File: step-templates/azure-linux-install-octopus-tentacle.json (hunk 1 above)
Comment (xtreampb) [reply step-templates/azure-linux-install-octopus-tentacle.json]: addresed in changeset 2ba51fc

**Review by harrisonmeister — APPROVED** [review]
LGTM

**Pull request merged**